src/api/publicationApi.ts

```diff
@@ -96,6 +97,26 @@ export const getPublication = async (id: string, dispatch: Dispatch) => {
   }
 };
 
+export const getMyPublications = async () => {
+  const url = `${PublicationsApiPaths.FETCH_MY_RESOURCES}`;
+  try {
+    const idToken = await getIdToken();
+    const response = await Axios.get(url, {
+      headers: {
+        Authorization: `Bearer ${idToken}`,
+      },
+    });
+    if (response.status === StatusCode.OK) {
+      console.log(response.data);
```

Comment:
console.log

Reply:
endpointet her er vel ikke klart, så det er kanskje bare bare en placeholder? isåfall greit nok

Reply:
ups.